app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt

```diff
@@ -92,17 +93,35 @@ class HomeMenu(
         }
     }
 
+    val syncedTabsItem = BrowserMenuImageText(
+        context.getString(R.string.synced_tabs),
+        R.drawable.ic_synced_tabs,
+        primaryTextColor
+    ) {
+        onItemTapped.invoke(Item.SyncedTabs)
+    }
+
+    private var signedInToFxa = false
     private fun getSyncItemTitle(): String {
         val authenticatedAccount = accountManager.authenticatedAccount() != null
         val email = accountManager.accountProfile()?.email
 
         return if (authenticatedAccount && email != null) {
+            signedInToFxa = true
             email
         } else {
             context.getString(R.string.sync_menu_sign_in)
         }
     }
 
+    val syncMenuItem = BrowserMenuImageText(
+        getSyncItemTitle(),
+        R.drawable.ic_synced_tabs,
+        primaryTextColor
+    ) {
+        onItemTapped.invoke(Item.SyncAccount(signedInToFxa))
```

Comment:
Having a function that queries the sync status as mentioned on https://github.com/mozilla-mobile/fenix/pull/19118/files#r616145969 will be more accurate to the latest sync status. I'm not sure if this state could change in the background, in case users invalidate their credentials from another device.

Reply:
If we check `accountManager.authenticatedAccount() != null` here we won't need `signedInToFxa` and it will be safer as we are checking the latest status :)